### ISSUE TITLE
[BUG] Dropdown menu separator not rendering

### ISSUE THREAD

**Issue by sistemicorp** [bokeh/bokeh#11964]
### Software versions

        Python version      :  3.8.10 (default, Nov 26 2021, 20:14:08) 
        IPython version     :  (not installed)
        Tornado version     :  6.1
        Bokeh version       :  2.4.1
        BokehJS static path :  /home/martin/.local/lib/python3.8/site-packages/bokeh/server/static
        node.js version     :  v16.13.2
        npm version         :  8.1.2

### Browser name and version

Chrome 97.0.4692.71 & Firefox v96.0, problem on both browsers

### Expected behavior

When creating a Dropdown widget, an entry of `None` causes a separator.

### Observed behavior

The separator is not visible, but the space for it is.  This can be seen in the bokeh Dropdown documentation.

![image](https://user-images.githubusercontent.com/49696383/150420655-e2b41154-f14e-4b4c-9fe7-28288172c4d0.png)

### Example code

```
When I make a small sample, the separator appears.  
However, in my larger application the separator is missing.

Given that the issue appears in the online documentation, 
I hope that can serve as an example.  Else I could try again to make a sample.

I noticed the problem when I moved to Bokeh v2.4.0.  
I am using v2.4.1.  I have not tested v2.4.2.
```

**Comment by bryevdv** [comment]
Thanks for the report @sistemicorp This appears to be resolved in `branch-3.0`

<img width="747" alt="Screen Shot 2022-01-20 at 14 04 26" src="https://user-images.githubusercontent.com/1078448/150429643-d69577cb-792d-44cd-9fc7-166096f53911.png">

Since it is unlikely that we will have another 2.4.x release at this point I am going to close this with `noaction`

**Comment by mattpap** [comment]
Since PR #11239 this is under visual test.
